Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -2419,11 +3521,15 @@ def main() -> None:
             "wab-get-mapping-of-user-group": client.get_mapping_of_user_group,
             "wab-edit-mapping-of-user-group": client.edit_mapping_of_user_group,
             "wab-delete-mapping-of-user-group": client.delete_mapping_of_user_group,
+            "wab-get-user-group-restrictions": client.get_user_group_restrictions,
+            "wab-add-restriction-to-usergroup": client.add_restriction_to_usergroup,
+            "wab-get-user-group-restriction": client.get_user_group_restriction,
+            "wab-edit-restriction-from-usergroup": client.edit_restriction_from_usergroup,
+            "wab-delete-restriction-from-usergroup": client.delete_restriction_from_usergroup,
+            "wab-get-version": client.get_version,
         }
 
-        deprecated = {
-            "wab-get-metadata-of-one-or-multiple-sessions": client.get_session_metadata,
-        }
+        deprecated: Dict[str, Any] = {}
```

Comment:
you can delete this one and if so, you can also delete the statment in line 3538: 
        elif command in deprecated:
            LOG(f"WARNING: use of deprecated command {command}")
            return_results(deprecated[command](args))

Reply:
done